smarctlide-monitoring/src/main/java/eu/smartclide/contexthandling/ServiceMain.java

```diff
@@ -0,0 +1,132 @@
+package eu.smartclide.contexthandling;
+/*-
+ * #%L
+ * smarctlide-monitoring
+ * %%
+ * Copyright (C) 2017 - 2019 ATB
+ * %%
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ * #L%
+ */
+
+import de.atb.context.monitoring.models.IMonitoringDataModel;
+import de.atb.context.services.AmIMonitoringService;
+import de.atb.context.services.IAmIMonitoringDataRepositoryService;
+import de.atb.context.services.IAmIMonitoringService;
+import de.atb.context.services.faults.ContextFault;
+import de.atb.context.services.wrapper.AmIMonitoringDataRepositoryServiceWrapper;
+import org.apache.cxf.endpoint.Server;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import de.atb.context.services.SWServiceContainer;
+import de.atb.context.services.manager.ServiceManager;
+import de.atb.context.tools.ontology.AmIMonitoringConfiguration;
+import org.springframework.util.ResourceUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Objects;
+import java.util.Properties;
+
+@SpringBootApplication
+public class ServiceMain {
+    private static final Logger logger = LoggerFactory.getLogger(ServiceMain.class);
+    private static Server server;
+    private static IAmIMonitoringService service;
+    private static AmIMonitoringDataRepositoryServiceWrapper monitoringDataRepository;
```

Comment:
Raw use of parameterized class 'AmIMonitoringDataRepositoryServiceWrapper'

Reply:
see the TODO comment